examples/demo-async-cluster.php

```diff
@@ -17,12 +17,18 @@
 
 require __DIR__ . '/../vendor/autoload.php';
 
+/**
+ * Start docker/local-3-node-cluster and run this script from host machine.
+ * This is because the cluster advertises as 127.0.0.1, which does not resolve
+ * to the event store in the PHP container, if you run it in Docker.
+ */
+
 Loop::run(function () {
     $builder = new ConnectionSettingsBuilder();
     $builder->setGossipSeedEndPoints([
-        new EndPoint('eventstore1', 2113),
-        new EndPoint('eventstore2', 2123),
-        new EndPoint('eventstore3', 2133),
+        new EndPoint('localhost', 2113),
```

Comment:
@pkruithof I needed to change this in order to make it work. Thoughts?

Reply:
It's the way I also was able to connect. It's too bad that you cannot run this from a PHP container yet, but we can work that out later. As long as we're able to test this, I'm fine with it.